### PR TITLE
Simplify integration test config

### PR DESCRIPTION
**Pull request by paulgb**
We don't need a separate test environment just to test auth, we can just pass in an auth environment variable (thanks @pretentious7 for figuring that out!)

### CODE REVIEW

File: crates/y-sweet-worker/wrangler.toml [hunk 1]
```diff
@@ -47,17 +47,3 @@ bindings = [
 binding = "Y_SWEET_DATA"
 bucket_name = "y-sweet-data-staging"
 preview_bucket_name = "y-sweet-data-dev"
-
-[env.auth-test]
-vars.WORKERS_RS_VERSION = "0.0.11"
```
Comment (pretentious7) [inline crates/y-sweet-worker/wrangler.toml:52]: this variable not needed anymore?

Comment (paulgb) [reply crates/y-sweet-worker/wrangler.toml:52]: the entire `auth-test` environment is no longer needed. Instead we just use the test env and pass an additional variable.

Comment (pretentious7) [reply crates/y-sweet-worker/wrangler.toml:52]: Yea but the version variable is not defined in the env vars

Comment (paulgb) [reply crates/y-sweet-worker/wrangler.toml:52]: It's defined on line 39 for `[env.test]`
image

The code that invoked `[env.auth-test]` no longer exists

image

**Pull request merged**